Failed to pull llamacpp-llama3-8b-instruct-bartowski-q5km:latest and failed to build the image

<!--
STOP -- PLEASE READ!

GitHub is not the right place for support requests.

If you're looking for help, check the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/)
or our [Mailing list](https://groups.google.com/g/kubernetes-sig-apps)

If the matter is security related, please disclose it privately via https://kubernetes.io/security/.
-->
Failed to pull image llamacpp-llama3-8b-instruct-bartowski-q5km:latest  with error :  
Failed to pull image "llamacpp-llama3-8b-instruct-bartowski-q5km:latest": initializing source docker://llamacpp-llama3-8b-instruct-bartowski-q5km:latest: reading manifest latest in docker.io/library/llamacpp-llama3-8b-instruct-bartowski-q5km: requested access to the resource is denied

when I try to build the leader image with : https://github.com/kubernetes-sigs/lws/blob/main/docs/examples/llamacpp/images/llamacpp-leader/Dockerfile  failed with error : 
```
-- Could NOT find CURL (missing: CURL_LIBRARY CURL_INCLUDE_DIR) 
CMake Error at common/CMakeLists.txt:95 (message):
  Could NOT find CURL.  Hint: to disable this feature, set -DLLAMA_CURL=OFF


-- Configuring incomplete, errors occurred!
See also "/workspace/llama.cpp/CMakeFiles/CMakeOutput.log".
See also "/workspace/llama.cpp/CMakeFiles/CMakeError.log".
Error: building at STEP "RUN cmake . -DGGML_RPC=ON -DBUILD_SHARED_LIBS=OFF -DGGML_DEBUG=1": while running runtime: exit status 1
```

--- a/docs/examples/llamacpp/images/llamacpp-leader/Dockerfile
+++ b/docs/examples/llamacpp/images/llamacpp-leader/Dockerfile
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1
 FROM debian:bullseye AS builder
 
-RUN apt-get update && apt-get install --yes git cmake g++
+RUN apt-get update && apt-get install --yes git cmake g++ libcurl4-openssl-dev
 
 RUN mkdir /workspace
 WORKDIR /workspace

--- a/docs/examples/llamacpp/images/llamacpp-worker/Dockerfile
+++ b/docs/examples/llamacpp/images/llamacpp-worker/Dockerfile
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1
 FROM debian:bullseye AS builder
 
-RUN apt-get update && apt-get install --yes git cmake g++
+RUN apt-get update && apt-get install --yes git cmake g++ libcurl4-openssl-dev
 
 RUN mkdir /workspace
 WORKDIR /workspace

--- a/site/content/en/docs/examples/llamacpp.md
+++ b/site/content/en/docs/examples/llamacpp.md
@@ -37,7 +37,7 @@ USE_EXISTING_CLUSTER=true KIND_CLUSTER_NAME=kind make test-e2e
 
 Finally, deploy the llama.cpp example to kind with [this script](https://github.com/kubernetes-sigs/lws/blob/main/docs/examples/llamacpp/dev/tasks/run-in-kind):
 ```shell
-./docs/examples/dev/tasks/run-in-kind
+./docs/examples/llamacpp/dev/tasks/run-in-kind
 ```
 
 After a lot of downloading and building (we build llama.cpp from source, and we download

